spring-cloud-common-security-config-web/src/main/java/org/springframework/cloud/common/security/ProviderRoleMapping.java

```diff
@@ -56,6 +57,21 @@ public ProviderRoleMapping(boolean mapOauthScopes, Map<String, String> roleMappi
 		this.roleMappings = roleMappings;
 	}
 
+	public boolean isParseOAuthPathParts() {
+		return parseOAuthPathParts;
+	}
+
+	/**
+	 * If set to false, OAuth scopes will not be treated as URIs during the role mapping and
```

Comment:
**[TinyNit]** Thanks for adding the docs (here and on the other method). 

I think it may be helpful to spell out the true/false conditions a bit w/ something like this: 

```java
Sets whether or not to treat OAuth scopes as URIs during the role mapping. 

When set to {@code true} the OAuth scope will be treated as a URI and the leading part will be ignored (eg. 'api://dataflow-server/dataflow.create' will result in 'dataflow.create').

When set to {@code false} the OAuth scope will be used as-is. This is useful in cases where the scope is not a URI and contains '/' leading characters. 


@param parseOauthPathParts  whether or not to treat OAuth scopes as URIs during the role mapping
```

Wdyt?

Reply:
Much better! Thanks for your time and the suggested doc - I applied it like this. 👍